yt_dlp/extractor/fourzerostudio.py

```diff
@@ -0,0 +1,108 @@
+from .common import InfoExtractor
+from ..utils import (
+    traverse_obj,
+    unified_timestamp,
+)
+
+
+class FourZeroStudioBaseIE(InfoExtractor):
+
+    @staticmethod
+    def _extract_uploader_info(nuxt_data):
+        uploader_info = traverse_obj(nuxt_data, ('ssrRefs', lambda _, v: v['__typename'] == 'PublicUser'), get_all=False)
+        return {
+            'uploader': uploader_info.get('username'),
+        }
```

Comment:
```suggestion
        return {
            'uploader': traverse_obj(nuxt_data, (
                'ssrRefs', lambda _, v: v['__typename'] == 'PublicUser', 'username'
            ), get_all=False),
        }
```
The function (and the baseIE) isn't even needed imo